torch/nn/quantized/modules/linear.py

```diff
@@ -95,7 +95,7 @@ class Linear(NNLinear):
     __constants__ = ['bias', 'in_features', 'out_features']
 
     def __init__(self, in_features, out_features, bias=True):
-        assert bias, 'nobias is not supported in Quantized Linear module yet'
+        # assert bias, 'nobias is not supported in Quantized Linear module yet'
```

Comment:
There is more work here in linear.py to handle the case of  bias=False. Currently, the code creates bias and calls fbgemm linear with self.bias as an argument.  Please modify forward so that it uses the fbgemm implementation with optional bias. For bias=False, the kernel should be called with no bias.
The change can be:
```
    def __init__(self, in_features, out_features, bias=True):
        super(Linear, self).__init__(in_features, out_features, bias)
        del self.weight
        if self.bias is not None:
            del self.bias
            qbias = torch._empty_affine_quantized(
            [out_features], scale=1, zero_point=0, dtype=torch.qint32)
            self.register_buffer('bias', qbias)
 ...
def forward(self,x):
       if bias is not None:
            Y_q = torch.ops.quantized.fbgemm_linear(
            x, self._packed_weight,
            self.bias,
            self.out_scale,
            self.out_zero_point)
     else:
            Y_q = torch.ops.quantized.fbgemm_linear(
            x, self._packed_weight,
            self.out_scale,
            self.out_zero_point)
        return Y_q

Reply:
Fixed. Thanks @raghuramank100 !